### DISCUSSION TITLE
Welcome to outreachy participants!

### DISCUSSION THREAD

**Discussion by tristanlatr** [LesFruitsDefendus/saskatoon-ng#126] [Outreachy]
Hello everyone,

I'm glad that our project is a part of outreachy for the year 2022 and I look forward to work with you!

You can review the project details here: https://www.outreachy.org/outreachy-may-2022-internship-round/communities/les-fruits-defendus/#getting-saskatoon-ng-ready

Please don't hesitate to open discussions here to ask anything or share some infos (note that selecting a category is required), I will reply as soon as I can.

**Comment by donaldte** [comment]
Good morinig to you sir my name is Tedom Donald student year three in  computer engineering in Cameroon.

and I am an Outreachy participants I have chosen to contribute to this magnificent project in order to be accepted as an intern at Outreachy.

Thanks so much

**Comment by donaldte** [comment]
I am installing all the tools, requirements and the project in question after that I will start working on the points you have given in the description.

**Reply by tristanlatr** [reply]
Hello @donaldte, 

Thanks for your interest into Saskatoon, 

As mentioned in this discussion: https://github.com/LesFruitsDefendus/saskatoon-ng/discussions/130

The issues stated in the outreachy project description are for the whole internship, so you shoun't start with those issues for the contribution period. For the short term, there are ton of other small improvements to do to Saskatoon in the list of gihub issues. 

Look forward for your contributions,

Talk to you later,

Tristan

**Comment by donaldte** [comment]
Hello sir I've already installed all the requirements and the projects itself. please if you want me to work on something please just point it out.
install screemshot
![install](https://user-images.githubusercontent.com/81464575/160225918-b993e597-1b7a-4a07-ab2f-cf8e9fe77c0b.png)

**Comment by mina-atef-00** [comment]
Glad to contribute.

**Comment by Austin-tawe** [comment]
Hello Mentor,

I am called Fad from Cameroon. I am an Outreachy Applicant for the March 2022 cohort. I am very much interested in the project "Getting Saskatoon-ng ready". I am excited about this new opportunity and looking forward to be a part of your wonderful team.
I Equally like to know more about the projects and any tips for beginners contribution and how to go about it.
Thanks

Best Regards!

**Reply by tristanlatr** [reply]
Hello @Austin-tawe,

Thanks for your interest into Saskatoon, the first step is really to setup a local instance of Saskatoon on your computer. The `INSTALL.md` file should describe the process to taht. I'll be glad to answer other questions :)

Talk to you later,

Tristan

**Comment by illyShelly** [comment]
Hello @tristanlatr and other Outreachies 🙂 

My name is ILONA. I studied Economics and worked in that field for a few years. However, my dream is to pursue to work in IT. 
I attended coding Bootcamp a few years ago (RoR)and was part of the Lisbon Startup programme. Unfortunately, Covid stopped everything. So now, I got this opportunity to come back into coding. I feel a bit overwhelmed but happy to be able to contribute to my first ever Open Source Project. Happy contribution 🤞

**Reply by tristanlatr** [reply]
Hello @illyShelly,

Congratulation! Hope this new path brings you the best.
Look forward for your contributions,

Tristan

**Comment by farhanmasud** [comment]
Hello @tristanlatr,

I'm Farhan Masud Aneek and I'm a Python/Django developer. My Outreachy initial application has been accepted and I'm very willing to work on this project.

I have made a PR and looking forward to contribute further to get this project up and running.

Best regards,
Farhan

**Reply by tristanlatr** [reply]
Hello @farhanmasud,

Thanks for your interest into Saskatoon and your PR!

I'll pull your branch and try it out in the next days, but just looking at the diff it looks great.

**Comment by minizzz0204** [comment]
Hello everyone,

I am Jasmine Zhan and I am an Outreachy applicant now, I am interested in working on this project.

Happy to cooperate with you guys!

Best,

Jasmine

**Reply by tristanlatr** [reply]
Hello @minizzz0204,

Thanks for your interest into Saskatoon,

Look forward for your contributions!

Tristan

**Comment by Rufaida98** [comment]
Hi everyone!!

My name is Rufaida Ali and i am an Outreachy applicant. I am interested in working in this project along with others!!

Best Regards.

Rufaida Ali

**Reply by tristanlatr** [reply]
Hi @Rufaida98,

Glad to hear that! Look forward for your contribution!

Tristan

**Comment by yokwejuste** [comment]
Greeting here! I'm Steve Yonkeu from Outreachy's initial applicants' list and I'm looking forward to working together with @lesfruits to contribute to this amazing project. Hopefully, I'll be able to give out the best I can.

**Reply by tristanlatr** [reply]
Look forward for your contribution @yokwejuste :)

**Reply by yokwejuste** [reply]
Please can you indicate the issues I can actually work on? If you can just tag them or link them I'll b glad! My system is set up already!

**Comment by ojaragodwin** [comment]
Hello, I am Godwin Ojara from Uganda, I am glad to be part of this magnificent  project, i just joined today and most of my queries have been answered in the Earlier asked questions

**Reply by tristanlatr** [reply]
Hello @ojaragodwin, 

Thanks for your interest into Saskatoon, look forward to you contributions :)

Talk to you later,

Tristan

**Comment by S-kus** [comment]
Hello Everyone! I am Saumya Kushwaha a 2nd year CSE B.Tech student at IIT BHU, India. My initial application for Outreachy May 2022 has been approved. I am already an open-source contributor and I find this project very interesting.
Looking forward to honing my skills by contributing to this project and learning from mentors and my colleagues.

**Comment by ojaragodwin** [comment]
Hello every one, my name is Godwin Ojara, an outreach applicant from
Uganda, I am happy to be part this project and very eager to contribute to
this cause

"Scientists investigate that which already exists; Engineers create that
which has never been."- Albert Einstein


Godwin Ojara.

Bsc. Electrical Engineering.

Makerere University, Kampala, Uganda.

Mobile: +256 785 700 395 / +256 700 167 245.

Email: ***@***.***

**Comment by ojaragodwin** [comment]
Hello every one, my name is Godwin Ojara, an outreach applicant from
Uganda, I am happy to be part this project and very eager to contribute to
this cause





"Scientists investigate that which already exists; Engineers create that
which has never been."- Albert Einstein


Godwin Ojara.

Bsc. Electrical Engineering.

Makerere University, Kampala, Uganda.

Mobile: +256 785 700 395 / +256 700 167 245.

Email: ***@***.***

**Comment by tristanlatr** [comment]
Hi everyone,

To know more about Saskatoon use cases, you can review the (outated) [guides](https://github.com/LesFruitsDefendus/guides): 

It will give you a good idea of the overhaul project usage and audience.

Talk to you later,

**Comment by singhavs** [comment]
Hello Mentor @lesfruits and other Outreachies 😄  
I am Aayushi Singh. I am an Outreachy Applicant for the March 2022 cohort. I am very much interested in the project "Getting Saskatoon-ng ready". I had locally set-up the environment and I am excited for this new opportunity and looking forward to be a part of your wonderful team.

Thanks
Best Regards
